config-templates/salesforce/readme.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+This directory contains a set of config files for modeling salesforce data. It also includes several common metrics calcuated from salesforce leads and opportunities. If you see a metric that's missing please open an issue in this repo.
+
+The config files are based on tables loaded into your data warehouse using fivetran and use the dimensions, identifier and measures names from these tables. If you are using a diffrent ETL tool, you may need to update the dimension, identifier or measurers, but the structure of the table should be the same.
```

Comment:
Nit: capitalize Fivetran
Nit: add `s` to list in the second sentence + oxford comma: "If you are using a different ETL tool, you may need to update the dimensions, identifiers, or measures"
sp: different